### PR TITLE
Update requirements with few changes

### PR DESCRIPTION
**Pull request by jwhitlock**
Update project requirements that do not require code changes. coverage / coveralls requires some configuration changes, but TravisCI will check most of them.

Work on [bug 1153288](https://bugzilla.mozilla.org/show_bug.cgi?id=1153288) is expected to find bugs in 3rd-party packages, so it is useful to get up to current releases before starting work.

### CODE REVIEW

**Comment by jwhitlock** [comment]
Upgraded Babel to 2.1.1

File: requirements.txt [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 
 # Django
-# 1.8 is waiting on django-simple-history
-Django==1.7.8  # rq.filter: >=1.7, <1.8
+Django==1.7.10  # rq.filter: >=1.7, <1.8
 
 # Better templates
 MarkupSafe==0.23
-Jinja2==2.7.3
+Jinja2==2.8
```
Comment (jezdez) [inline requirements.txt]: Are you sure that works in practice? We had a few issue with enforced encoding in kuma.

**Comment by jezdez** [comment]
I haven't checked all dependencies for every change, but I trust you have and are fine with the result rwc+

File: requirements.txt (hunk 1 above)
Comment (jwhitlock) [inline requirements.txt]: There's only a few pages with non-ASCII content, which still work after this change (using public links, tested locally):

https://browsercompat.herokuapp.com/api/v1/maturities/1
https://browsercompat.herokuapp.com/browse/maturities/2
https://browsercompat.herokuapp.com/importer/issues/inline_text (fancy punctuation)
https://browsercompat.herokuapp.com/importer/726 (select alternate language)
https://browsercompat.herokuapp.com/api/v1/view_features/730?format=html&lang=ja

I'll watch for issues when it goes to production.

Comment (jwhitlock) [inline requirements.txt]: Pushed to prod, pages still render w/o errors.

**Pull request merged**